src/sidebar/components/test/new-note-btn-test.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+var angular = require('angular');
+
+var events = require('../../events');
+var util = require('../../directive/test/util');
+
+describe('newNoteBtn', function () {
+  var $rootScope;
+  var sandbox = sinon.sandbox.create();
```

Comment:
I wondered if it would be a problem that the sandbox is created once but restored multiple times (in `afterEach`). The example in the SinonJS docs at http://sinonjs.org/releases/v4.1.1/sandbox/ do this though, so I'm assuming that's fine.